gradle/java-module.gradle

```diff
@@ -107,35 +107,19 @@ dependencies {
 		testRuntime( libraries.hana )
 	}
 
-
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// Java 9 ftw!
 	if ( JavaVersion.current().isJava9Compatible() ) {
-		// The JDK used to run Gradle is Java 9+, and we assume that that is the same
-		//		JDK for executing tasks
-		compile( libraries.jaxb_api )
-		compile( libraries.jaxb_runtime )
-		compile( libraries.jaxb_xjc )
-		compile( libraries.jaxb2_basics )
-		compile( libraries.jaxb2_basics_ant )
+		// In order for generated by jpamodelgen to be compiled successfully,
+		// we need the @Generated annotation to be in the classpath,
+		// and it's no longer in the JDK starting with JDK9.
```

Comment:
It's not exactly true. It has been moved.

Reply:
Fixed